### PR TITLE
Introduce Real time instance type class and RHEL 9 real time preference

### PR DESCRIPTION
**Pull request by lyarwood**
/hold

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Added a realtime instancetype class and the rhel.9.realtime preference that are suited for realtime applications like Oslat.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +252,15 @@ o1.micro  |  1  |  1Gi
...
 o1.small  |  1  |  2Gi
 o1.xlarge  |  4  |  16Gi
+rt1.2xlarge  |  8  |  32Gi
+rt1.4xlarge  |  16  |  64Gi
+rt1.8xlarge  |  32  |  128Gi
+rt1.large  |  2  |  8Gi
+rt1.medium  |  1  |  4Gi
+rt1.micro  |  1  |  1Gi
+rt1.nano  |  1  |  512Mi
+rt1.small  |  1  |  2Gi
```
Comment (0xFelix) [inline README.md:262]: Will we ever see workloads with such a small footprint for this specialized use case?

File: instancetypes/rt/1/rt1.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      realtime applications.
+    instancetype.kubevirt.io/displayName: "Realtime"
+  labels:
+    instancetype.kubevirt.io/class: "realtime"
+    instancetype.kubevirt.io/icon-pf: "pficon-server-group"
+    instancetype.kubevirt.io/version: "1"
+    instancetype.kubevirt.io/vendor: "kubevirt.io"
+    instancetype.kubevirt.io/dedicatedCPUPlacement: "true"
+    instancetype.kubevirt.io/hugepages: "true"
+    instancetype.kubevirt.io/isolateEmulatorThread: "true"
```
Comment (0xFelix) [inline instancetypes/rt/1/rt1.yaml:21]: Missing a label for numa and realtime?

Comment (0xFelix) [reply instancetypes/rt/1/rt1.yaml:20]: Added missing labels

File: README.md [hunk 3]
```diff
@@ -256,6 +282,14 @@ o1.micro  |  1  |  1Gi
 o1.nano  |  1  |  512Mi
 o1.small  |  1  |  2Gi
 o1.xlarge  |  4  |  16Gi
+rt1.2xlarge  |  8  |  32Gi
+rt1.4xlarge  |  16  |  64Gi
+rt1.8xlarge  |  32  |  128Gi
+rt1.large  |  2  |  8Gi
+rt1.medium  |  1  |  4Gi
+rt1.micro  |  1  |  1Gi
```
Comment (ksimon1) [inline README.md:290]: shouldn't be here 4Gi memory?

Comment (ksimon1) [reply README.md:290]: the same for small

Comment (0xFelix) [reply README.md:290]: These sizes are intended to be even smaller than `rt1.medium`. Therefore they have a different ratio.

**Review by codingben — APPROVED** [review]
Nice work!

/lgtm

**Pull request merged**